vumi/transports/smpp/smpp_transport.py

```diff
@@ -499,6 +504,12 @@ def _check_stop_throttling(self):
         """
         self._unthrottle_delayedCall = None
 
+        if not self.service.is_bound():
+            # We don't have a bound SMPP connection, so try again later.
+            log.msg("Can't check throttling while unbound, trying later.")
+            self.check_stop_throttling(self.get_static_config().throttle_delay)
```

Comment:
This is only relevant during a reconnect (we can't be throttled until we've sent at least one message), and it has more or less the same behaviour as retrying a throttled message and getting another RTHROTTLED response.